okhttp-clients/src/main/java/com/palantir/remoting3/okhttp/OkHttpClients.java

```diff
@@ -54,6 +54,16 @@
      */
     private static final ExecutorService executionExecutor = Tracers.wrap(new Dispatcher().executorService());
 
+    /**
+     * Shared dispatcher with static executor service.
+     */
+    private static final Dispatcher dispatcher = createDispatcher();
```

Comment:
maybe move the initialization to a `static {}` block so it's clear that the createDispatcher method shouldn't be used?